src/utils.ts

```diff
@@ -521,3 +531,16 @@ export function hasValidDleq(proof: Proof, keyset: MintKeys): boolean {
 
 	return true;
 }
+
+export function tokenToRawToken(token: string | Token): Uint8Array {
+	if (typeof token === 'string') {
```

Comment:
I would prefer this to only accept token and `outsource` conversion to the implementer. I think the readability improves if functions only do one (or close to one) thing.

Reply:
Yeah that makes sense to me. I will change it accordingly